example-mlem-get-started/code/README.md

```diff
@@ -0,0 +1,40 @@
+# MLEM Get Started
+
+This is an auto-generated repository for use in MLEM
+[Get Started](https://mlem.ai/doc/get-started). It is a step-by-step quick
+introduction into basic MLEM concepts.
+
+- The `main` branch contains the results of following the Get Started. Results
+  of each step can be found in git tags.
+- The `dvc` branch contains the results of the
+  [MLEM+DVC tutorial](http://mlem.ai/doc/use-cases/dvc).
+
+🐛 Please report any issues found in this project here -
+[example-repos-dev](https://github.com/iterative/example-repos-dev).
+
+## Installation
+
+Python 3.7+ is required to run code from this repo.
+
+To be able to commit and push your results please fork and then clone this repo
+
+```console
+$ gh repo fork https://github.com/iterative/example-mlem-get-started
```

Comment:
It's really bad practice to depend on `gh`, we should prefer using vanilla `git` commands as were discussing before.